.circleci/config.yml

```diff
@@ -248,28 +296,34 @@ workflows:
           filters:
             branches:
               only: /^release\/.*/
+
       - release_build:
           filters:
             branches:
               only: master
+
       - deploy_last_build:
           requires:
             - release_build
           filters:
             branches:
               only: master
+
+      - notify_about_merge_to_master:
```

Comment:
this is where we setup `notify_about_merge_to_master` job to be executed on each master merge